modules/examiner/jsx/examiner.js

```diff
@@ -0,0 +1,252 @@
+import FilterForm from 'FilterForm';
+import formatColumn from './columnFormatter';
```

Comment:
`columnFormatter` should be included as a function within this class as per #3938.